index.html

```diff
@@ -15,5 +15,17 @@
 	</div>
 	<button>Menu</button>
 	</nav>
+	
+	<header>
+		<div class="img">
+			<div class="about">
+				<h1>About Me</h1>
+				<p> Hello, I'm Diana. I am training to become a full-stack web developer. I started training with iO in August 2021. By the end I will have a diverse set of skills, ranging from desgin to HTML, CSS, Javascript all the way to PHP.</p>
+				<p>I was first introduced to the world of coding on a little island in Malaysia. From there I dipped my toes in the waters of coding, which has landed me to where I am today.   
+				</p>
+			</div>
+	</div>
```

Comment:
indent this one to match the div at the top

Reply:
done